eurocropsml/acquisition/config.py

```diff
@@ -11,6 +11,24 @@
 
 logger = logging.getLogger(__name__)
 
+S2_BANDS = [
+    "01",
+    "02",
+    "03",
+    "04",
+    "05",
+    "06",
+    "07",
+    "08",
+    "8A",
+    "09",
+    "10",
+    "11",
+    "12",
+]
```

Comment:
Is the order only important for `S1_BANDS` or also for `S2_BANDS`? Maybe we can clarify by having a comment on both lists separately.

Reply:
Good point! It is indeed important for both. I'll add the comment to the S2 bands.